src/features/MentorPage/components/MentorList/index.tsx

```diff
@@ -13,17 +14,21 @@ type Props = {
 
 const MentorList: React.FC<Props> = ({ setVisibleCard, mentors }) => {
   const { isMobile } = useGetLayoutMode();
+  const isMentorsEmpty: boolean = mentors.length === 0;
```

Comment:
Maybe a bit redundant to define a constant for a very simple condition :thinking:

Reply:
I disagree a bit. Having a variable, even for this simple condition makes the render-return a bit easier to read imo. 😛

Reply:
🤔 i also think for readability its nicer to have a variable

though it does not take too long to refactor this to not have the variable